source/ada/lsp-ada_handlers.ads

```diff
@@ -268,6 +268,10 @@ private
       Completion_Snippets_Enabled : Boolean := False;
       --  True if the client supports completion snippets
 
+      Use_Completion_Snippets : Boolean := True;
```

Comment:
should we set this to False by default? so that the testsuite tests the default IDE behavior? (unless tests specifically activate snippets)